src/app/home/home.component.ts

```diff
@@ -75,6 +75,10 @@ export class HomeComponent implements AfterViewInit, OnDestroy {
             () => {
                 (document.getElementById('editor') as any)!.contentEditable =
                     false;
+                    const btndown = document.querySelectorAll('.card-header button') as any;
+                    for(let btnsdown of btndown){
+                    btnsdown?.setAttribute('disabled', 'disabled');
+                }
```

Comment:
These changes should be fine, few minor things, is "as any" needed here? Also, consider using forEach https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach instead of a usual for-loop. Finally, maybe .disabled=true might be better than setAttribute('disabled', 'disabled');, have a look and let me know.

Reply:
Can you also extract this code to a method and call that method here? I'd say the code for disabling the editor deserves its own method (perhaps called `disableEditor`).

Reply:
Reviewing it again. Will check what can I do.